### ISSUE TITLE
A11y of theme selection page

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2373]
There are a few accessibility issues of note on the theme section:

- [x] All `.theme-thumbnail` images are missing `alt` attributes, let alone alt text. Alt text that describes the basic design and distinguishing characteristics of each theme should be added.
- [ ] Focus can be brought to focusable elements (such as links) within selectable `.theme` divs, but the divs themselves are not focusable elements. Thus, themes cannot be selected via keyboard in the current setup. I would imagine a `button` element or role could be subbed in here to rectify this?
- [ ] The color contrast of a theme marked for selection should be bumped to at least 3:1 (currently at 1.2:1).
- [x] All theme names are tagged as headings, which are currently `<h4>` and should be `<h2>` for logical semantics. However, headings may not make sense here at all if button function is brought in to the selectable themes, and proper markup is added to achieve a similar purpose with the theme name.
